recipes/bolt/meta.yaml

```diff
@@ -0,0 +1,143 @@
+
+package:
+  name: bolt-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
+  sha256: 58a8818c60e6627064f312dbf46c02d9949956558340938b71cf731ad8bc0813
+  # matching https://github.com/conda-forge/llvmdev-feedstock/tree/main/recipe/patches,
+  # since we rebuild llvmdev here (until https://github.com/llvm/llvm-project/pull/97130)
+  patches:
+    - patches/0001-pass-through-QEMU_LD_PREFIX-SDKROOT.patch
+    # backports patch for issues with LLJIT, see
+    # https://github.com/llvm/llvm-project/commit/122ebe3b500190b1f408e2e6db753853e297ba28
+    - patches/0002-ORC-Use-EPC-bootstrap-symbols-to-communicate-eh-fram.patch
+
+build:
+  number: 0
+  merge_build_host: false
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - libcxx {{ cxx_compiler_version }}  # [osx]
+  host:
+    - libcxx {{ cxx_compiler_version }}  # [osx]
+    # needs aarch/ppc/arm (re)build of conda-forge/backtrace-feedstock
+    - backtrace                          # [unix and x86]
+    - libxml2
+    - zlib
+    - zstd
+
+outputs:
+  # Contains bolt support libraries
+  - name: libbolt-devel
+    script: install_bolt.sh   # [unix]
+    script: install_bolt.bat  # [win]
+    build:
+      activate_in_script: true
+      always_include_files:
+        # Must overwrite CMake targets from llvmdev
+        {% set prefix = "Library/" if win else "" %}
+        - {{ prefix }}lib/cmake/llvm/LLVM-Config.cmake
+        - {{ prefix }}lib/cmake/llvm/LLVMConfig.cmake
+        - {{ prefix }}lib/cmake/llvm/LLVMConfigVersion.cmake
+        - {{ prefix }}lib/cmake/llvm/LLVMExports-release.cmake
+        - {{ prefix }}lib/cmake/llvm/LLVMExports.cmake
+
+    requirements:
+      build:
+        - {{ stdlib('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+        - libcxx {{ cxx_compiler_version }}  # [osx]
+      host:
+        # ensure we don't pick up stuff already packaged in llvmdev
+        - llvmdev {{ version }}
+      run:
+        # necessary due to shared CMake metadata
+        - llvmdev {{ version }}
+      run_constrained:
+        - llvm        {{ version }}
+        - llvmdev     {{ version }}
+        - clang       {{ version }}
+        - clang-tools {{ version }}
+    test:
+      commands:
+      {% for each_lib in [
+          "Core", "Passes", "Profile", "Rewrite", "RuntimeLibs",
+          "TargetAArch64", "TargetX86", "Utils"
+      ] %}
+        - test -f $PREFIX/lib/libLLVMBOLT{{ each_lib }}.a               # [unix]
+        - if not exist %LIBRARY_LIB%\LLVMBOLT{{ each_lib }}.lib exit 1  # [win]
+      {% endfor %}
+        # only on linux-64
+        - test -f $PREFIX/lib/libbolt_rt_hugify.a                       # [linux64]
+        - test -f $PREFIX/lib/libbolt_rt_instr.a                        # [linux64]
```

Comment:
These two should be in `bolt` as they are used at runtime of `bolt`. The others are only needed if bolt is used as a library.

Reply:
I'm not sure (between the two open threads) what you want me to do now. Here you say
> The others are only needed if bolt is used as a library.

But in the other you say
> If there are no include files, then we might as well drop `libbolt-devel` as users can't use those libraries

I understood that we'll move `libbolt_rt_{hugify,instr}` to `bolt`. But for the rest, do we want to:
* ship them as `libbolt-devel`
* ship them as part of `bolt`
* not ship them at all until a use-case comes up

Reply:
3 is preferred. 1 if not.

Reply:
OK, I found the bolt headers: https://github.com/llvm/llvm-project/tree/main/bolt/include/bolt; no idea why they're not getting installed.

Reply:
Also, I guess we can restrict this to linux until anything changes upstream w.r.t. https://github.com/llvm/llvm-project/issues/72205